pkg/tasks/client/client.go

```diff
@@ -3,9 +3,11 @@ package client
 import (
 	"context"
 
+	"github.com/content-services/content-sources-backend/pkg/config"
 	"github.com/content-services/content-sources-backend/pkg/tasks"
 	"github.com/content-services/content-sources-backend/pkg/tasks/queue"
 	"github.com/google/uuid"
+	"golang.org/x/exp/slices"
```

Comment:
one small thing and this is good to go! this import should be "slices" since it's now included in the standard library